proto/backend.proto

```diff
@@ -25,6 +25,7 @@ message DataSourceInstanceSettings {
   map<string,string> decryptedSecureJsonData = 9;
   int64 lastUpdatedMS = 10;
   string uid = 11;
+  string type = 12;
```

Comment:
To make my earlier comment clearer. Unnecessary to send the same information twice over the wire when `pluginId` already part of `PluginContext` below.